### PR TITLE
add navigation component

### PR DESCRIPTION
**Pull request by iRohitGaur**

### CODE REVIEW

File: pages/nav.css [hunk 1]
```diff
@@ -51,9 +51,10 @@ nav {
   gap: 2px;
   grid-area: nav;
   margin: 0 3rem;
-  padding-top: 4rem;
+  padding: 4rem 0;
   font-size: 1.6rem;
   position: relative;
+  overflow: scroll;
```
Comment (zoef-yt) [inline pages/nav.css:57]: make overflow: auto so it only appear when needed.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: pages/nav.css (hunk 1 above)
Comment (iRohitGaur) [reply pages/nav.css:57]: I have hidden the scrollbar

Comment (varun13169) [reply pages/nav.css:57]: +1

**Review by varun13169 — APPROVED** [review]

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Comment by archit7448** [comment]
good to merge

**Review by archit7448 — APPROVED** [review]

**Pull request merged**